.github/workflows/pr-workflow.yml

```diff
@@ -0,0 +1,48 @@
+name: PR Workflow
+run-name: ${{ github.actor }} pull request
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types:
+      - opened
```

Comment:
The commits done to the feature branch haven't triggered new runs of github actions. I suppose it is because it runs only for types: opened.

I suppose the type -edited could be added here as well:

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request